services/task-svc/internal/bed/bed.go

```diff
@@ -0,0 +1,220 @@
+package bed
+
+import (
+	"common"
+	"context"
+	pb "gen/proto/services/task_svc/v1"
+	"github.com/google/uuid"
+	zlog "github.com/rs/zerolog/log"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"hwgorm"
+	"task-svc/internal/room/models"
+)
+
+type ServiceServer struct {
+	pb.UnimplementedBedServiceServer
+}
+
+func NewServiceServer() *ServiceServer {
+	return &ServiceServer{}
+}
+
+type Base struct{}
```

Comment:
?

Reply:
_`/* Empty for later extension and use */`_